ProjectMarsLanguageFeature/Pages/LanguagesPage.cs

```diff
@@ -11,7 +11,7 @@
 namespace ProjectMarsLanguageFeature.Pages
 {
     public class LanguagesPage
-            {
+    {
         LanguageLevelOptions levelOptionObj = new LanguageLevelOptions();
```

Comment:
can you pleae implement the pom pattern see for reference https://docs.specflow.org/projects/specflow/en/latest/ui-automation/Selenium-with-Page-Object-Pattern.html

Reply:
I noticed you tried to break down the components but you will learn this properly, I am curious to know who has taught you this way can you please message me on slack about this as this is good but can be improved

Reply:
Sure, I will through reference and implement accordingly